frontend/src/components/app/App.js

```diff
@@ -241,19 +241,19 @@ const App = ({userID, usersByID, logOut, get}) => {
                     {env !== 'PROD' && <span style={{ color: 'red', fontSize: '14px' }}>&nbsp;{env}</span>}
                   </div>
                 </Title>
-                { // Display a spinner while the initial data is being fetched at startup 
+                { // Display a spinner while the initial data is being fetched at startup
                   !isInitialized &&
                     <div style={{display: 'flex', flexDirection: 'column', justifyContent: 'center'}} className="App__spin">
                       <Spin size="small" indicator={loadingIcon}/>
                     </div>
                 }
               </div>
             </div>
-            {isLoggedIn &&
+            {/* {isLoggedIn &&
               <div className='App__jumpBar'>
                 <JumpBar />
               </div>
-            }
+            } */}
```

Comment:
If it's not useful, it might as well be removed.

Reply:
i dont think this is the right assessment over this component

Reply:
the component needs to be configured correctly if you guys want to use it. 
it would help the user jump from one page to to another, is this something freezeman would need?

Reply:
It was requested multiple times by David (id think he likes it) i never use it myself but i don't see why we would remove a working component ... The upgrade breaks it ? Maybe the css can be replaced with some component style ? I really would prefer we don't start removing stuff just because we upgrade antd ...

Reply:
As far as i can see as long as it is not commented out it works fine ...

Reply:
the upgrade did make this component look out of place when i first started to work on this task, i decided to remove it for the meantime while i worked on the other more breaking parts of the css. i've tried to use it but i couldn't get anywhere with it. i keep receiving 5XX server error codes. 

I propose to open a ticket for it on redmine and investigate it

Reply:
disregard last line of previous comment, no ticket needed